doc/example/patterns_retrieve_cref.cpp

```diff
@@ -17,6 +17,7 @@ namespace
 
     class my_class
     {
+        base_class& b;
```

Comment:
It’s a bit sad that this shows up in the docs given it’s an implementation detail of the test and it’s not needed to explain the «pattern».

Reply:
On the other hand it shows users what is done with the ctor argument. I could cheat and make it a global pointer to get it out of sight.

Reply:
I think we should really try to keep the code the shows up in the documentation minimal.
I know when I read code in docs I like it to be simple and straight-forward, I’m sure we all do given the choice :)

Reply:
Ok, removed that